src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/PropertyGridTests.cs

```diff
@@ -4064,4 +4078,70 @@ public new bool VScroll
 
         public new void WndProc(ref Message m) => base.WndProc(ref m);
     }
+
+    private class ErroredCombo
+    {
+        [TypeConverter(typeof(ExpandableObjectConverter))]
+        public virtual ComboPath Path { get; set; } = new ComboPath() { };
+
+        public class ComboPath
+        {
+            [NotifyParentProperty(true)]
+            [TypeConverter(typeof(StringConverter))]
+            public string ComboName { get; set; }
+
+            public override string ToString() => $"{{ComboName='{ComboName}'}}";
+        }
+    }
+
+    private class MySite : ISite
+    {
+        private IComponentChangeService _componentChangeService = new ComponentChangeService();
+        public MySite(IComponent component, IContainer container)
+        {
+            Component = component;
+            Container = container;
+        }
+
+        public IComponent Component { get; private set; }
+
+        public IContainer Container { get; private set; }
+
+        public bool DesignMode => true;
```

Comment:
```suggestion
        public bool DesignMode => false;
```

to make this test a little simpler (execute less code)